SEPADebit/src/main/java/com/braintreepayments/api/SEPADebitMandateType.java

```diff
@@ -0,0 +1,19 @@
+package com.braintreepayments.api;
+
+import androidx.annotation.StringDef;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+/**
+ * Mandate type for the SEPA Debit request.
+ */
+@Retention(RetentionPolicy.SOURCE)
+@StringDef({
+        SEPADebitMandateType.RECURRENT,
+        SEPADebitMandateType.ONE_OFF
+})
+public @interface SEPADebitMandateType {
+    String RECURRENT = "RECURRENT";
+    String ONE_OFF = "ONE_OFF";
+}
```

Comment:
random: we could technically go with enums, or I forget if we were gonna use enums in the next major version.

Reply:
Yeah that's legit. Enums used to be a huge memory hog but Google fixed it at some point.